README.md

```diff
@@ -46,6 +46,7 @@ Flutter ile ilgili Türkçe makaleler kategorilerine göre ayrılmış halde bur
 - [Flutter HEALTHO UI 2](https://medium.com/flutter-t%C3%BCrkiye/flutter-healtho-ui-2-96077afaea3e) - Veli Bacık
 - [Flutter HEALTHO UI](https://medium.com/flutter-t%C3%BCrkiye/flutter-healtho-ui-ad72c823e8b2) - Veli Bacık
 - [Container, Column, Row…](https://medium.com/flutter-t%C3%BCrkiye/container-column-row-89b31f988605?source=collection_detail----f3fe55e37b59-----24-----------------------) - Zeki Çıplak
+- [Animated Selection Slide](https://github.com/sbilketay/animated_selection_slide) - Sezgin Bilgetay
```

Comment:
Ilk PR acip katkida bulunan ilk kisi oldugun icin cok tesekkur ederim. 👍 Bu bir kutuphane oldugu icin, asagida yeni bir kutuphane kismi olusturup UI ile ilgili kisma ekleyecegim. Ama bunu merge etmekte sakinca yok :) 
__________________________________________________________________________________________________
Thank you for being the first person to open up a PR and making a contribution! I will merge this one but move it to library section. I will move this down and create a section for it. But there is no harm to merge it at the moment :)

Reply:
Boyle faydali bir repo olusturdugunuz icin asil ben tesekkur ederim 😊